### PR TITLE
Format per linter suggestions

### PR DESCRIPTION
**Pull request by cottsay**
Again, the motivation is to help align the ROS 1 package with the ROS 2 package to minimize the drift.

I also spotted some dangling `import rospy` that could be removed after the logging change.

Don't worry about merging this into the ROS 2 development branch - I'll take care of that when the time comes.

### CODE REVIEW

File: src/rqt_reconfigure/filter_children_model.py [hunk 1]
```diff
@@ -108,23 +107,22 @@ def _filter_row_recur(self, src_row, src_parent_qmindex):
...
 
             # Set all subsequent treenodes True
-            logging.debug(' FCModel.filterAcceptsRow src_row={}'.format(
-                            src_row) +
-                           ' parent row={} data={}'.format(
-                              src_parent_qmindex.row(),
-                              src_parent_qmindex.data()) +
-                           ' filterRegExp={}'.format(regex))
+            logging.debug(
+                ' FCModel.filterAcceptsRow src_row={}'.format(src_row) +
```
Comment (gonzodepedro) [inline src/rqt_reconfigure/filter_children_model.py]:
```suggestion
                ' FCModel.filterAcceptsRow src_row={} parent row={} data={} filterRegExp={}' .format(
                         src_row, src_parent_qmindex.row(), src_parent_qmindex.data()), regex))
```

Comment (gonzodepedro) [inline src/rqt_reconfigure/filter_children_model.py]: how about:
`                ' FCModel.filterAcceptsRow src_row={} parent row={} data={} filterRegExp={}'.format(
                          src_row, src_parent_qmindex.row(), src_parent_qmindex.data()), regex)`

File: src/rqt_reconfigure/node_selector_widget.py [hunk 2]
```diff
@@ -463,16 +470,17 @@ def _test_sel_index(self, selected, deselected):
 
         if selected.indexes() > 0:
             logging.debug('sel={} par={} desel={} sel.d={} par.d={}'.format(
-                                 index_current, index_parent, index_deselected,
-                                 index_current.data(Qt.DisplayRole),
-                                 index_parent.data(Qt.DisplayRole),)
-                                 + ' desel.d={} cur.item={}'.format(
-                                 None,  # index_deselected.data(Qt.DisplayRole)
-                                 curr_qstd_item))
+                index_current, index_parent, index_deselected,
```
Comment (gonzodepedro) [inline src/rqt_reconfigure/node_selector_widget.py]: Same here

File: src/rqt_reconfigure/node_selector_widget.py [hunk 3]
```diff
@@ -463,16 +470,17 @@ def _test_sel_index(self, selected, deselected):
...
+                    curr_qstd_item
+                ))
         elif deselected.indexes():
             logging.debug('sel={} par={} desel={} sel.d={} par.d={}'.format(
-                                 index_current, index_parent, index_deselected,
-                                 None, index_parent.data(Qt.DisplayRole)) +
-                           ' desel.d={} cur.item={}'.format(
-                                 index_deselected.data(Qt.DisplayRole),
-                                 curr_qstd_item))
+                                index_current, index_parent, index_deselected,
```
Comment (gonzodepedro) [inline src/rqt_reconfigure/node_selector_widget.py]: and here

File: src/rqt_reconfigure/filter_children_model.py (hunk 1 above)
Comment (cottsay) [reply src/rqt_reconfigure/filter_children_model.py]: I ran into the 79-character line length limit, but I tried to break it up in a more sane way. Let me know if that looks better to you.

**Pull request merged**